### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by niqdev** [SIMULATAN/XServerDocker#1]
Hi,
sorry for the straight question, I saw that this project was very recently updated and I'm wondering, is this actually working??

I've been trying to build an ubuntu/debian/kali image recently and none of them are working. I either get a grey or a black screen. I have an old image that used to work, but when I try to build a fresh one from sources, "something" is broken and I can't figure out what it is. There are no clear errors and all I get is a black screen. My image is very similar to this one, that's why I'm asking.

I've excluded the vncviewer client, literally took the latest nightly build of [VncViewer](http://tigervnc.bphinz.com/nightly/)

This is your project for example
```bash
# top shell
java -jar VncViewer-1.12.80.jar localhost:5900

# bottom shell
docker-compose up
```
![Screenshot from 2022-09-21 09-26-23](https://user-images.githubusercontent.com/9458643/191454776-3619e6ff-93f6-46bf-9f3f-3c51e857cb63.png)

Could you please tell me if this project is working? At least I have some hope! BTW, I maintain [edgelevel/alpine-xfce-vnc](https://github.com/edgelevel/alpine-xfce-vnc) and this is actually working smoothly!

Thanks in advance for the feedback

**Comment by SIMULATAN** [comment]
Hello, and thanks for taking your time to checkout my project!

I believe it is actually working just fine, this is how it looks like when the X server is running without a DE or WM. This repo was made to be as lightweight as possible, and thus only features a running minimal X server with VNC.

I made a second image that has DWM and auto-starts a terminal (st in my case), in which case you _can_ see something just fine.
This second image is on my other PC tho, so I can't send you that right now unfortunately. I'll just try to replicate it on this device, I'll let you know if I got a demo working.

In a nutshell, you can just start a window manager with it's shell command (`dwm &` for example) _after_ you started the X server. After that, you can also start a terminal, like this: `st &`.

Thanks a lot for your repo, I'll definitely check that out!

**Comment by SIMULATAN** [comment]
Hey, it's me again!

I managed to get a super simple example with dwm and st going, see [this repo (SIMULATAN/XServerDocker-dwm-st)](https://github.com/SIMULATAN/XServerDocker-dwm-st)

**Comment by niqdev** [comment]
Awesome, thanks for the quick feedback. I'll check it out later and I'll give an update!

**Comment by niqdev** [comment]
that actually works 🥳 , you need to change the package name from `st` to `stterm` though! Thanks a lot for the feedback!

**Comment by SIMULATAN** [comment]
> that actually works partying_face , you need to change the package name from `st` to `stterm` though! Thanks a lot for the feedback!

Oh really? It works fine for me :thinking:
![screenshot](https://user-images.githubusercontent.com/63104422/191499032-3494ac4c-2b09-4bc1-83b3-44333a350494.png)
running `apt show st` also shows 0 results, so maybe it was packaged with dwm?
Also, I just ran `docker-compose build` instead of `up`, and now it doesn't build anymore because, as you said, the package isn't found?? Quite confused, but hey, it's working, so i'll take that as a success :)

Glad you could sort it out tho, I'll update the repo rq.

**Comment by niqdev** [comment]
Thanks, left a comment in the commit you've linked me with the fix
